koku/api/report/aws/aws_query_handler.py

```diff
@@ -92,6 +89,12 @@ def _format_query_response(self):
 
         return output
 
+    def _set_tag_filters(self, filters):
```

Comment:
This is streamlining and simplifying the call-chain. 

We were passing `no_tag_query` up from this child class through the parent classes via kwargs purely to set this filter in the parent class to be consumed by this child class. 

Instead, we should do the right thing and override the method to insert the filter right here in this class. There's no need to involve the parent class at all.